core/coreapi/unixfs.go

```diff
@@ -254,3 +259,29 @@ func (api *UnixfsAPI) lsFromLinks(ctx context.Context, ndlinks []*ipld.Link, set
 func (api *UnixfsAPI) core() *CoreAPI {
 	return (*CoreAPI)(api)
 }
+
+func (api *UnixfsAPI) AppendMetadata(metaMap map[string]interface{}, opts ...options.UnixfsAddOption) error {
```

Comment:
This is unused, this is for future feature?

Reply:
Yes. That is right. For Robin's requirement to use metadata.